pr-builder.rst

```diff
@@ -0,0 +1,93 @@
+.. post:: July 14, 2019
+   :tags: gsoc, pr-builder, continuous-documentation, feature
+   :author: Saadmk11
+   :location: DHA
+
+Autobuild docs for Pull Requests
+================================
+
+Building Documentation for pull/merge request has always been one of the most requested feature of Read the Docs.
+This feature will allow users to build documentation for pull/merge request automatically using ``webhooks``
+and get status notification to the Providers Status API (e.g. Github Status API).
+This will help users to check if the documentation build passed
+and also how the documentation looks before merging it to master.
+
+As a student of Google Summer of Code (GSoC) 2019, I (`Maksudul Haque`_) was tasked with building this feature.
+The Main goal of my project was to make it possible to build documentation when ever a pull/merge request was created,
+and send build status notification to the Providers Status API.
+
+Google Summer of Code
+=====================
+
+Google Summer of Code is a global program where students work with an open source organization
+on a 3 month programming project. **Autobuild docs for Pull Requests** was one of the project ideas on
+Read the Docs `Project Ideas`_ for Google Summer of Code 2019. I applied for the project and got accepted.
+
+I have started working on this project from May 2019.
+All of my work on this project can be seen in the `Pull Request Builder Board`_.
+
+Background
+==========
+
+Many of our users wanted a way to visualize the documentation update that was made in a pull/merge request.
+They also wanted to know the status of a build ``(passed/failed)`` before merging the pull/merge request to master.
+This would allow users to have more confidence on the pull request
+and make the pull/merge request less likely to break the documentation after merging.
+So, In reply to this Read the Docs `core team`_ selected **Autobuild docs for Pull Requests**
+as one of the projects of Google Summer of Code 2019.
+
+Pull Request Builder Features
+=============================
+
+Currently I have implemented these features and working on more upcoming features.
+Some of the major features are as following:
+
+- **Creating External Versions:** We create an ``External Version`` when we receive ``pull_request`` ``opened/reopened``
```

Comment:
These should probably not be monospaced. They aren't code.

![Screen Shot 2019-07-15 at 11 17 42 AM](https://user-images.githubusercontent.com/185043/61238754-2edc5980-a6f2-11e9-9369-b4777607f1b5.png)

Reply:
okay